packages/wallets/src/components/WalletsAccordion/WalletsAccordion.scss

```diff
@@ -56,6 +56,10 @@
         transition: all 0.3s ease;
         cursor: pointer;
         z-index: 10;
+        display: inline-block;
```

Comment:
This looks brand new. Was this ment to be part of the refactor?

Reply:
@markwylde-deriv This is for accessibility, Without this only the blue part is click able, I'm increasing the click area to the green one.

<img width="276" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/3a6646b3-9d92-440e-aaca-c62b6690d1a2">